<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="register_btn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>登记</string>
    </property>
   </widget>
   <widget class="QComboBox" name="register_com_box">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>住户信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>楼栋信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>住房信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>物业费信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>停车费信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>车位信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>维修信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>维修基金</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>员工信息</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="query_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>501</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="query_editor">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="update_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>更新</string>
    </property>
   </widget>
   <widget class="QComboBox" name="update_com_box">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>住户信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>楼栋信息</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="del_com_box">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>510</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>住户信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>楼栋信息</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="del_btn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QComboBox" name="query_com_box">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>住户信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>楼栋信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>住房信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>物业费信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>停车费信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>车位信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>维修信息</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>维修基金</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>员工信息</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="count_fee_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>210</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>计算物业、停车费</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sta_pro_fee_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>统计物业费</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sum_parking_space_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>320</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>计算停车位总数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sum_resident_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>380</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>计算小区总户数</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>726</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
